src/TestCentric/testcentric.gui/SettingsPages/SettingsPage.cs

```
@@ -143,7 +143,7 @@ protected override void OnLoad(EventArgs e)
 
 			if ( !DesignMode )
 			{
-                Settings = (ParentForm as SettingsDialogBase)?.Settings;
+                Settings = new UserSettings((ParentForm as SettingsDialogBase)?.Settings);
```

It can be the case that `ParentForm` will always be a `SettingsDialogBase`, but I would rather fail early here than sending null to `UserSettings` (and in turn let `SettingsGroup` fail with a NRE).

---

It's a good point. Will do.